### DISCUSSION TITLE
CSS not working in Github Pages

### DISCUSSION THREAD

**Discussion by jasminehuelgas** [community/community#137598] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Guys! can you help me out my website is not working properly in github pages. The CSS is not working.

**Comment by ThisIs-Developer** [comment]
### Troubleshooting CSS Issues on GitHub Pages

1. **Check CSS File Path:**
   - Ensure the CSS file path is correct and relative to the HTML file.

2. **Check for Case Sensitivity:**
   - GitHub Pages is case-sensitive; make sure the file names and paths match exactly.

3. **Inspect Network Requests:**
   - Use browser Developer Tools to check if the CSS file is being loaded correctly.

4. **Verify File Upload:**
   - Ensure the CSS file is properly committed and pushed to the repository.

5. **Check for Cache Issues:**
   - Clear browser cache or try a hard refresh (Ctrl+F5 or Cmd+Shift+R).

6. **Review HTML Link Tags:**
   - Ensure `<link>` tags are correct: 
     ```html
     <link rel="stylesheet" href="styles.css">
     ```

7. **Check GitHub Pages Settings:**
   - Confirm your GitHub Pages settings are pointing to the correct branch and folder.

### If this helps, please mark as answer!😴

**Comment by LiteBrite82** [comment]
@jasminehuelgas 
Start with resolving the errors in the console. Change your `href`s from `http` to `https`
<img width="1893" alt="Screenshot 2024-09-05 at 2 31 14 PM" src="https://github.com/user-attachments/assets/657b5cce-a6d1-45b4-86e6-7eb89418abae">

**Reply by ThisIs-Developer** [reply]
@jasminehuelgas 
 - **Fix**: Update the resource URLs to use HTTPS. For example, change:
     ```html
     <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
     ```
     to:
     ```html
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
     ```

**Reply by jasminehuelgas** [reply]
Hello! So I followed the steps that you advised me, but the CSS still isn't working. I really don't know what to do anymore.

**Comment by Raykesh-Soneka** [comment]
Hi Jasmine Huelgas @jasminehuelgas ,

I look into the repo : https://github.com/jasminehuelgas/jasminehuelgas.github.io
And in this [site](https://jasminehuelgas.github.io/), the css still don't work.

For the following file in lines 7 and 9, you have to put the s in http, like @ThisIs-Developer showed you.
- [index.html](https://github.com/jasminehuelgas/jasminehuelgas.github.io/blob/9227c9640baf8f7ba622c7bec291a614d5aace6c/index.html#L7)
- [contact.html](https://github.com/jasminehuelgas/jasminehuelgas.github.io/blob/9227c9640baf8f7ba622c7bec291a614d5aace6c/contact.html#L7)
- [services.html](https://github.com/jasminehuelgas/jasminehuelgas.github.io/blob/9227c9640baf8f7ba622c7bec291a614d5aace6c/services.html#L7)
- [about.html](https://github.com/jasminehuelgas/jasminehuelgas.github.io/blob/9227c9640baf8f7ba622c7bec291a614d5aace6c/about.html#L7)
- [Resume.html](https://github.com/jasminehuelgas/jasminehuelgas.github.io/blob/9227c9640baf8f7ba622c7bec291a614d5aace6c/Resume.html#L7)
- [portfolio.html](https://github.com/jasminehuelgas/jasminehuelgas.github.io/blob/9227c9640baf8f7ba622c7bec291a614d5aace6c/portfolio.html#L7)

## CSS [![My Skills](https://skillicons.dev/icons?i=css)](https://skillicons.dev)
line 7 to 9 :
```CSS
<link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
<script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>
```
Into
```CSS
<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>
```

> [!NOTE]
> You can copy paste the change.

**Reply by jasminehuelgas** [reply]
Hello! So I followed the steps that you advised me, but the CSS still isn't working. I really don't know what to do anymore.

**Comment by LiteBrite82** [comment, marked as answer]
Answered in subsequent post: https://github.com/orgs/community/discussions/143525
